src/slacko.mli

```diff
@@ -149,6 +149,14 @@ type message_length_error = [
   | `Msg_too_long
 ]
 
+(** When posting a message with attachments, you may receive this error if you
```

Comment:
I'll reword that. The docs state that you'll only get an error if you post more than 100 attachments, but that you shouldn't post more than 20.